legion/engine/core/platform/platform.hpp

```diff
@@ -420,7 +420,7 @@ LEGION_GCC_SUPPRESS_WARNING("-Wc++14-compat")
 #   define L_ALWAYS_INLINE
 #endif
 
-#if (defined(LEGION_WINDOWS) && !defined(LEGION_WINDOWS_USE_CDECL)) || defined (DOXY_INCLUDE)
+#if (defined(LEGION_WINDOWS) && !defined(LEGION_WINDOWS_USE_CDECL)) || defined(DOXY_INCLUDE)
     /**@def LEGION_CCONV
      * @brief the calling convention exported functions will use in the args engine
```

Comment:
not actually the line you changed but you should probably ctrl+f through the project sometime to clean up args engine references

Reply:
yeah i mean we have to do so when we rename to rythe again as well